docs/user-guides/user-guide-gui.md

```diff
@@ -0,0 +1,83 @@
+# Use the Web Interface to Find and Pull Images
+
+> :point_right: Using a browser, you can browse a **zot** repository and download container images and artifacts.
+
+If a zot registry is built with the optional `zui` package, the registry has a web interface. 
+
+## Access the registry
+
+> :pencil2: Depending on the security configuration of the zot registry, you might need to authenticate before being given access.
+
+The initial page of the registry displays a sampling of the most popular images and recently updated images. To view all available images in the registry, click the `View all` link in the upper right of the page. 
+
+![screen-initial](../assets/images/screen-initial.jpg){width="600"}
+
+> :bulb: Wherever an image name appears, two icons follow the name, indicating the results of the vulnerabiity scan and the signature status. These icons are described in [Icons and their meanings](#icons).
+
+## Find an image
+
+To assist you in finding images specific to your requirements, the `View all` page provides several sorting options along with faceted navigation in addition to a general search box.
```

Comment:
I think our tentative name for the page is the 'Explore' page, as it can also be accessed through the search bar in the header. More thoughts on the official name?

Reply:
We don't title the page, but the URL is _registry_/explore.   In the text, we could call it the `/explore` page.